system/audit-logs/values.yaml

```
@@ -56,7 +56,10 @@ fluent_audit_container:
       host: none
   alerts:
     enabled: false
-    prometheus: infra-collector
+    # Keys = directory names in alerts/ and aggregations/
+    prometheus:
+      infra-collector: infra-collector
+      thanos-ruler: metal
```

Maybe we can restructure this to not having to rely on parsing the name
```suggestion
    prometheus:
      - name: infra-collector
        type: prometheus
      - name: metal
        type: thanos-ruler
```